validation/src/test/scala/hmda/validation/rules/lar/validity/V455Spec.scala

```diff
@@ -0,0 +1,39 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V455Spec extends LarEditCheckSpec {
+  import Gen.posNum
+
+  property("Succeeds when Applicant Ethnicity is not 1, 2, or 3") {
```

Comment:
I intentionally chose to use a generator for that value (instead of just using 4) because I wanted to reflect the actual behavior of the edit check. This edit check is not enforcing valid ethnicity values (that happens in a different edit check), so the LAR might have a totally bogus value in that field. I wanted to make it clear how this edit behaves, even when the ethnicity value isn't valid.